vmm/src/cpu.rs

```diff
@@ -1213,7 +1213,7 @@ impl CpuManager {
 
         // This reuses any inactive vCPUs as well as any that were newly created
         for vcpu_id in self.present_vcpus()..desired_vcpus {
-            let vcpu = Arc::clone(&self.vcpus[vcpu_id as usize]);
+            let vcpu = Arc::clone(&self.vcpus[vcpu_id as u32]);
```

Comment:
This won't work - you can't subscript with u32. Did you run clippy over all the commits? (or better - set up the pre-commit hook from CONTRIBUTING.md to do it for you.)